llama_index/vector_stores/types.py

```diff
@@ -56,18 +57,23 @@ class VectorStoreQueryMode(str, Enum):
     LINEAR_REGRESSION = "linear_regression"
 
 
-@dataclass
-class ExactMatchFilter:
+class ExactMatchFilter(BaseModel):
     key: str
     value: Union[str, int, float]
 
 
-@dataclass
-class MetadataFilters:
+class MetadataFilters(BaseModel):
```

Comment:
are we implementing and and or conditions later?

Reply:
Ya we can extend , though not sure how useful this really would be